add json support

klay-ke opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vesoft-inc/nebula-java?pullRequest=358) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vesoft-inc/nebula-java?pullRequest=358) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vesoft-inc/nebula-java?pullRequest=358) it.</sub>

Nicole00 commented:
Great! please sign the CLA and format your code style.

Nicole00 commented on client/src/main/java/com/vesoft/nebula/client/graph/net/Session.java:
```diff
@@ -100,6 +100,99 @@ public synchronized ResultSet execute(String stmt) throws IOErrorException {
+     *                   "data":[
+     *                     {
+     *                       "row":row-data,
+     *                       "meta":metadata]
```
breaken format?

klay-ke replied:
nice catch, hard copied from Yichen's pr, might be his is also broken

HarrisChu commented:
how abort to separate two commits, one is for generate code, one is for the others.
then we can review others directly by filter.

![image](https://user-images.githubusercontent.com/1726587/134839730-bf588f64-ad3e-46ae-b646-e2c29f8fc93b.png)

HarrisChu commented on client/src/test/java/com/vesoft/nebula/client/graph/data/TestDataFromServer.java:
```diff
@@ -390,4 +392,66 @@ public void testErrorResult() {
+                    .getString("errorMsg");
+
+            // check errorcode
+            Assert.assertEquals(errCode, "E_SEMANTIC_ERROR");
```
errCode should be an int?
`1009` instead of `E_SEMANTIC_ERROR` ?

HarrisChu replied:
talked with @Aiee  offline, just follow neo4j schema.

Aiee replied:
For consistency, let's use int for errorCode here.

HarrisChu reviewed (dismissed):
LGTM except errorCode.

Nicole00 commented:
Refresh the thrift files according to pr https://github.com/vesoft-inc/nebula/pull/2926. Change the datatype of json response's errorCode to Int.

klay-ke commented:
> According to pr [vesoft-inc/nebula#2926](https://github.com/vesoft-inc/nebula/pull/2926), the datatype of json response's errorCode is changed to Int.

Build and saw no changes since last commit

Nicole00 commented on client/src/test/java/com/vesoft/nebula/client/graph/data/TestDataFromServer.java:
```diff
@@ -390,4 +392,65 @@ public void testErrorResult() {
+                    .getString("message");
+
+            // check error code
+            Assert.assertEquals(code, -1009);
```
you can use ErrorCode.getValue instead of -1009， in case the code changes.

Aiee reviewed (approved):
LGTM.
